source/onboard/cloud-sso-saml.rst

```diff
@@ -18,10 +18,10 @@ For more information about SAML, see `this article from Varonis <https://www.var
 
 Mattermost officially supports Okta, OneLogin, and Microsoft ADFS as the identity providers (IDPs), please see links below for more details on how to configure SAML with these providers.
 
- - `Okta SAML Configuration <https://docs.mattermost.com/deployment/sso-saml-okta.html>`_
```

Comment:
Thanks for this! Just as a note, we have redirects set up for the majority of the links we've changed so not all of them need to be replaced. It definitely does make a difference though and we appreciate it. :)